docs/tutorial/part-eight/index.md

```diff
@@ -1,94 +1,94 @@
 
-![Lighthouse audit results](./lighthouse-audit-results.png)
+![Lighthouse ऑडिट परिणाम](./lighthouse-audit-results.png)
 
-As you can see, Gatsby's performance is excellent out of the box but you're missing some things for PWA, Accessibility, Best Practices, and SEO that will improve your scores (and in the process make your site much more friendly to visitors and search engines).
+जैसा की आप देख सकते हैं, Gatsby का परफॉरमेंस आउट ऑफ़ दी बॉक्स बहुत अच्छा है, लेकिन इसमें आप PWA, एक्सेसिबिलिटी, बेस्ट प्रैक्टिसेज और SEO को मिस कर रहे होंगे जो स्कोर्स को अच्छा करता है (और इस प्रक्रिया में आपकी साइट विज़िटर्स और सर्च इंजनों के लिए बहुत अधिक फ्रेंडली बनाएंगे)।
 
-## Add a manifest file
+## मैनिफेस्ट फाइल ऐड करना
 
-Looks like you have a pretty lackluster score in the "Progressive Web App" category. Let's address that.
+लगता है "Progressive Web App" केटेगरी में आपका स्कोर बहुत मंदा है। आइये उसके बारे बात करें।
 
-But first, what exactly _are_ PWAs?
+लेकिन पहले, PWAs है क्या चीज़?
 
-They are regular websites that take advantage of modern browser functionality to augment the web experience with app-like features and benefits. Check out [Google's overview](https://developers.google.com/web/progressive-web-apps/) of what defines a PWA experience.
+यह मामूली वेबसाइट्स है जो आधुनिक ब्राउज़र की कार्यक्षमता का फायदा उठाकर वेब अनुभव को अप्प-लाइक विशेषतायें एवं फायदों के साथ ऑग्मेंट करता है। [Google का अवलोकन](https://developers.google.com/web/progressive-web-apps/) को देखिये जो यह दिखाता है की एक सही PWA अनुभव को क्या परिभाषित करता है।
 
-Inclusion of a web app manifest is one of the three generally accepted [baseline requirements for a PWA](https://alistapart.com/article/yes-that-web-project-should-be-a-pwa#section1).
+एक वेब ऐप मैनिफ़ेस्ट का शामिल होना आम तौर पर स्वीकृत तीन [PWA की आधारभूत आवश्यकताओं](https://alistapart.com/article/yes-that-web-project-should-be-a-pwa#section1) में से एक है।
 
-Quoting [Google](https://developers.google.com/web/fundamentals/web-app-manifest/):
+[Google](https://developers.google.com/web/fundamentals/web-app-manifest/) का उद्धरण करते हुए:
 
-> The web app manifest is a simple JSON file that tells the browser about your web application and how it should behave when 'installed' on the user's mobile device or desktop.
+> वेब अप्प मैनिफेस्ट एक सरल JSON फाइल है जो ब्राउज़र को आपके वेब एप्लीकेशन के बारे में बताता है और यह भी बताता है की उसका व्यवहार कैसा होना चाहिए जब वह यूजर के मोबाइल डिवाइस या डेस्कटॉप पर इन्सटाल्ड हो।
 
-[Gatsby's manifest plugin](/packages/gatsby-plugin-manifest/) configures Gatsby to create a `manifest.webmanifest` file on every site build.
+[Gatsby की मैनिफेस्ट प्लगइन](/packages/gatsby-plugin-manifest/) हर साइट बिल्ड में Gatsby को कॉन्फ़िगर करके एक `manifest.webmanifest` फाइल बनाती है। 
```

Comment:
`फाइल बनाती है। ` => `फाइल बनाती है।`

remove extra space

Reply:
I don't find any extra space here.

Reply:
extra space is at the end of the line after `है।`

Reply:
extra space is at the end of the line after `है।`

Reply:
> extra space is at the end of the line after `है।`

correct